src/amo/actions/reviews.js

```diff
@@ -584,3 +586,30 @@ export const updateAddonReview = ({
     payload: { body, errorHandlerId, rating, reviewId },
   };
 };
+
+export const ABORTED: 'aborted' = 'aborted';
+export const SAVED_RATING: 'saved-rating' = 'saved-rating';
+export const SAVED_REVIEW: 'saved-review' = 'saved-review';
+export const STARTED_SAVE_RATING: 'started-save-rating' = 'started-save-rating';
+export const STARTED_SAVE_REVIEW: 'started-save-review' = 'started-save-review';
+
+export type FlashMessageType =
+  | typeof ABORTED
+  | typeof SAVED_RATING
+  | typeof SAVED_REVIEW
+  | typeof STARTED_SAVE_RATING
+  | typeof STARTED_SAVE_REVIEW;
+
+export type FlashReviewMessageAction = {|
+  type: typeof FLASH_REVIEW_MESSAGE,
+  payload: { message: FlashMessageType | void },
```

Comment:
can `void` just be added to `flashMessageType` above?

Reply:
Well, huh. I initially wanted it like that but Flow gave me some errors that I couldn't figure out. I just tried it again and Flow is happy 🤷‍♂️  The error must have been related to calling code that is now fixed.